src/pieces/__init__.py

```diff
@@ -1,4 +1,8 @@
 import importlib.metadata
 
-__version__ = importlib.metadata.version("pieces") # This gets the current version which is set in pyproject.toml
+try:
```

Comment:
Nice, was it causing issues before? May be good to add an issue to add a link to download when we have it available

Reply:
Yes I got an error before that happen when you fork the repo and run the code the main file instead of downloading it using pip